handlers/customer/handler.go

```diff
@@ -1,25 +1,25 @@
 package customer
 
 import (
-	"database/sql"
+	"customerApi/stores"
 	"encoding/json"
 	"github.com/gorilla/mux"
 	"io"
 	"log"
 	"net/http"
 
-	"customerApi/drivers"
 	"customerApi/models"
```

Comment:
sort imports and do not use mixed caps for package, directory and file names

Reply:
done